README.md

```diff
@@ -1,5 +1,5 @@
 [![PyPI version](https://badge.fury.io/py/semeio.svg)](https://badge.fury.io/py/semeio)
-[![Build Status](https://travis-ci.com/equinor/semeio.svg?branch=master)](https://travis-ci.com/equinor/semeio)
+[![Actions Status](https://github.com/equinor/semeio/workflows/CI/badge.svg)](https://github.com/equinor/semeio/actions?query=workflow=CI)
```

Comment:
According to this https://docs.github.com/en/free-pro-team@latest/actions/managing-workflow-runs/adding-a-workflow-status-badge#using-a-workflow-name, the following line would be enough for the CI badge
![Actions Status](https://github.com/equinor/semeio/workflows/CI/badge.svg)
Should I change it? And do we want to see the status of passing tests only on master or on every branch?

Reply:
So the second part of that badge is the link if you actually press the badge, which I think is nice to have as a shortcut to the CI. So I would say keep that link as well.

The main reason for having badges for other branches tends to be with a slightly different workflow than us. We tend to work on master, while other projects might for example have a feature branch as well as master. Then it makes sense to have a different badge for different branches, in our case, I would say master is enough.